doc/manipulating_images/manipulating_images.rst

```diff
@@ -132,6 +132,14 @@ Accessing individual volumes in 4D images
      creates a new image using an existing reference image for the
      metadata.
 
+|
+
+.. topic:: **Examples**
```

Comment:
For the record, there is a way to include all the examples using a particular function (although it may only be in sphinx-gallery master) which looks like this:
```
.. include:: modules/generated/numpy.linspace.examples
.. raw:: html

        <div style='clear:both'></div>
```

The documentation: http://sphinx-gallery.readthedocs.io/en/stable/advanced_configuration.html#establishing-local-references-to-examples.